src/Microsoft.Framework.Caching.Abstractions/CancellationTokenTrigger.cs

```diff
@@ -3,31 +3,33 @@
 
 using System;
 using System.Threading;
+using Microsoft.Framework.Primitives;
 
 namespace Microsoft.Framework.Caching
 {
-    public class CancellationTokenTrigger : IExpirationTrigger
+    /// <summary>
+    /// A <see cref="IChangeToken"/> implementation using <see cref="CancellationToken"/>.
+    /// </summary>
+    public class CancellationTokenTrigger : IChangeToken
```

Comment:
1. Should we rename this? `CancellationChangeToken` \ `CancellationTokenBasedChangeToken`?
2. FileProvider uses this type. Should we move this to Common or have FileSystem replicate it?